tests/core/OperationProcessor.spec.ts

```diff
@@ -693,10 +694,16 @@ describe('OperationProcessor', async () => {
           operationIndex: 2
         };
 
+        // Intentionally failing update patches.
         spyOn(DocumentComposer, 'applyUpdateOperation').and.throwError('Expected test error');
```

Comment:
Instead of directly throwing error, we can change some attribute in the document, then throw. The expect on line 707 can then be a test to make sure documents come out the same.

Reply:
Thanks for the great suggestion!